document/core/exec/modules.rst

```diff
@@ -798,6 +798,8 @@ Once the function has returned, the following steps are executed:
 
 2. Pop :math:`\val_{\F{res}}^m` from the stack.
 
+3. Pop the frame :math:`F` from the stack.
```

Comment:
Can you add a corresponding assertion before this step, analogous to step 18 of instantiation?